lib/tbot/tbot_test.go

```diff
@@ -169,24 +167,24 @@ func TestBot_Run_CARotation(t *testing.T) {
 	rotate(ctx, t, log, teleportProcess(), types.RotationPhaseInit)
 	// TODO: These sleeps allow the client time to rotate. They could be
 	// replaced if tbot emitted a CA rotation/renewal event.
-	time.Sleep(time.Second * 15)
+	time.Sleep(time.Second * 30)
```

Comment:
Could we use https://pkg.go.dev/github.com/stretchr/testify/require#Eventually here?

Reply:
Not really unfortunately. Because the rotation isn't an immediate process, something which started checking immediately could mean that we'd be checking it's healthy before the rotation has started. This would potentially cause a more confusing failure at the next step.

We have plans to introduce an events emitter for the `tbot.Bot` which will allow us to do this a bit more intelligently in the near future.

Reply:
So, that would provide half the story, but still wouldn't provide much insight as to whether or not the Bot had received and started processing the update events from the server. This becomes a little more complex since the Bot applies a bit of a debounce (it waits ten seconds after receiving the first event to trigger a CA rotation caused renewal, so that we don't renew multiple times in a row when all of the CAs are rotated at once).

Reply:
Yeah, it seems tricky to validate
Thank you for the explanation